src/explore-education-statistics-common/src/modules/table-tool/components/LocationFiltersForm.tsx

```diff
@@ -153,12 +156,19 @@ const LocationFiltersForm = ({
       initialValues={initialFormValues}
       validationSchema={validationSchema}
     >
-      {({ formState, getValues, reset }) => {
+      {({ formState, getValues, reset, setError }) => {
         const showError =
           !formState.isValid &&
           (Object.keys(formState.touchedFields).length ||
             formState.isSubmitted);
 
+        if (showError && !get(formState.errors, 'locations')) {
+          setError('locations', {
+            type: 'min',
+            message: errorMessage,
+          });
+        }
```

Comment:
This doesn't work because we want to show an error if there is nothing selected in any group. With your suggested change the error on blur is assigned to the group, e.g. to `locations.localAuthority`, so if you then select  a location in a different group the error is not removed. Using `setError` to create a general error for `locations` was the only way I could find to handle this correctly.

I've made the changes to `showError` and `initialFormValues` that you suggested.

Reply:
Good point!

Given it another think and how about using `trigger` instead:
```tsx
<FormProvider>
  {({ ..., trigger }) => {
    const showError =
      !formState.isValid &&
      (Object.keys(formState.touchedFields).length > 0 ||
        formState.isSubmitted);

    if (showError && !formState.errors.locations) {
      trigger('locations');
    }
  }}
</FormProvider>
```

This means we don't can use Yup correctly as intended, rather than manually set the error.

Then in the `FormFieldset`, we can do:

```tsx
<FormFieldset
  error={
    showError
      ? getErrorMessage(formState.errors, 'locations')
      : ''
  }
>
```

Reply:
yep, `trigger` works, cheers 👍